File: servidor/models/tienda.py
from sqlalchemy import Column, Text, Boolean
from db.database import Base

class Tienda(Base):
    __tablename__ = 'tiendas'

    codigo = Column(Text, primary_key=True)
    direccion = Column(Text)
    ciudad = Column(Text)
    provincia = Column(Text)
    habilitada = Column(Boolean, default=True)

File: proveedor/src/main/java/proveedor/models/Tienda.java
package proveedor.models;


import jakarta.persistence.*;
import lombok.Data;

@Entity
@Table(name = "tiendas")
@Data
public class Tienda {

    @Id
    @Column(name = "codigo", nullable = false)
    private String codigo;

    @Column(name = "direccion")
    private String direccion;

    @Column(name = "ciudad")
    private String ciudad;

    @Column(name = "provincia")
    private String provincia;

    @Column(name = "habilitada", nullable = false)
    private boolean habilitada = true;
}
